plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -193,6 +193,12 @@ export class EventPipelineRunner {
             }
         }
 
+        // if the event is a groupidentify event, we should always skip person processing entirely. We
+        // don't allow use of $set and friends on groupidentify events
+        if (event.event === '$groupidentify') {
```

Comment:
Note that above, on line 154, we still drop the event if person-processing is disabled by the user (using the relevant property) - I think not doing that has implications for pricing that I'm not confident we want to change (since it would allow use of groups in personless mode)